### PR TITLE
Update SharePoint Online Unlicensed

### PR DESCRIPTION
**Pull request by salarson**
Can we move this line to it's own item and specifically call out that removing a license does not kick off the my site cleanup job in OneDrive?

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/manage/remove-licenses-from-users.md [hunk 1]
```diff
@@ -143,7 +143,9 @@ When you use the **Licenses** page to unassign licenses, you unassign licenses f
 
 - You can also [delete user accounts](../add-users/delete-a-user.md) that have been assigned a license to make their license available to other users. When you delete a user account, their license is immediately available to be assigned to someone else.
 
-- When a license is removed from a user, data that is associated with that  account is held for 30 days. After the 30-day grace period, the data is deleted and can't be recovered. Files that are saved in OneDrive or SharePoint Online aren't deleted. For more information, See [OneDrive retention and deletion](https://docs.microsoft.com/onedrive/retention-and-deletion)
+- When a license is removed from a user, data that is associated with that  account is held for 30 days. After the 30-day grace period, the data is deleted and can't be recovered.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/remove-licenses-from-users.md]:
```suggestion
- When a license is removed from a user, data that is associated with that account is held for 30 days. After the 30-day grace period, the data is deleted and can't be recovered.
```

File: microsoft-365/admin/manage/remove-licenses-from-users.md [hunk 2]
```diff
@@ -143,7 +143,9 @@ When you use the **Licenses** page to unassign licenses, you unassign licenses f
...
+
+- Note:  Files that are saved in OneDrive for Business aren't deleted unless the user is deleted from the Microsoft 365 admin center or is removed through Active Directory synchronization. For more information, See [OneDrive retention and deletion](https://docs.microsoft.com/onedrive/retention-and-deletion)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/remove-licenses-from-users.md]:
```suggestion
- Note that files saved in OneDrive for Business aren't deleted unless the user is deleted from the Microsoft 365 admin center or is removed through Active Directory synchronization. For more information, see [OneDrive retention and deletion](https://docs.microsoft.com/onedrive/retention-and-deletion).
```

File: microsoft-365/admin/manage/remove-licenses-from-users.md [hunk 3]
```diff
@@ -143,7 +143,9 @@ When you use the **Licenses** page to unassign licenses, you unassign licenses f
...
 
    Also, the user's mailbox is no longer searchable by using an eDiscovery tool such as Content Search or Advanced eDiscovery when the license is removed. For more information, see the "Searching disconnected or de-licensed mailboxes" section in [Content Search in Office 365](https://docs.microsoft.com/office365/securitycompliance/content-search#searching-disconnected-or-de-licensed-mailboxes).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/remove-licenses-from-users.md]:
```suggestion
- Also, the user's mailbox is no longer searchable by using an eDiscovery tool such as Content Search or Advanced eDiscovery when the license is removed. For more information, see the "Searching disconnected or de-licensed mailboxes" section in [Content Search in Office 365](https://docs.microsoft.com/office365/securitycompliance/content-search#searching-disconnected-or-de-licensed-mailboxes).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**